### PR TITLE
Gradle Challenge

### PR DESCRIPTION
**Pull request by davidgzu**
Gradle Challenge, hackerank exercises into Gradle project

### CODE REVIEW

File: DavidGonzalezZumaya/Gradle/src/main/java/ConcurrencyFormatter/ConcurrencyFormatter.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package ConcurrencyFormatter;
```
Comment (luisod78) [inline DavidGonzalezZumaya/Gradle/src/main/java/ConcurrencyFormatter/ConcurrencyFormatter.java:1]: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: DavidGonzalezZumaya/Gradle/src/main/java/JavaIntToString/DoNotTerminate.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package JavaIntToString;
```
Comment (luisod78) [inline DavidGonzalezZumaya/Gradle/src/main/java/JavaIntToString/DoNotTerminate.java:1]: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: DavidGonzalezZumaya/Gradle/src/main/java/JavaIntToString/JavaIntToString.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package JavaIntToString;
```
Comment (luisod78) [inline DavidGonzalezZumaya/Gradle/src/main/java/JavaIntToString/JavaIntToString.java:1]: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: DavidGonzalezZumaya/Gradle/src/main/java/JavaStaticInitializerBlock/JavaStaticInitializerBlock.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package JavaStaticInitializerBlock;
```
Comment (luisod78) [inline DavidGonzalezZumaya/Gradle/src/main/java/JavaStaticInitializerBlock/JavaStaticInitializerBlock.java:1]: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: DavidGonzalezZumaya/Gradle/src/main/java/JavaStaticInitializerBlock/JavaStaticInitializerBlock.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    static int breadth = scanner.nextInt();
+    static int height = scanner.nextInt();
+
+    static {
+        try {
+            if (breadth <= 0 || height <= 0) {
+                flag = false;
+                throw new Exception(EXCEPTION_MESSAGE);
+            }
+        } catch (Exception e) {
```
Comment (luisod78) [inline DavidGonzalezZumaya/Gradle/src/main/java/JavaStaticInitializerBlock/JavaStaticInitializerBlock.java:24]: Prefer specific Exceptions
https://dzone.com/articles/9-best-practices-to-handle-exceptions-in-java

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by lrod-glb — APPROVED** [review]

**Review by globant-bootcamp — APPROVED** [review]

**Pull request merged**